### ISSUE TITLE
Harvest data from package.json for nodejs projects

### ISSUE THREAD

**Issue by rw251** [hermes-hmc/workflow#36]
Projects using NodeJS will have a package.json file which may contain data such as:
- author
- contributors
- title
- description
- license
- dependencies

This data could be harvested.

**Comment by sdruskat** [comment]
Yes, this is definitely something that would go in the manifest-like harvesting, together with other manifest types. We're in the process of setting up a better roadmap for us (and you) to track the types of sources we're implementing, and in what order.
